src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx

```diff
@@ -140,9 +142,14 @@ export function AccessCodeTable({
       <ContentHeader onBack={onBack} />
       <TableHeader>
         <div className='seam-left'>
-          <TableTitle>
-            {t.accessCodes} <Caption>({filteredAccessCodes.length})</Caption>{' '}
-          </TableTitle>
+          {title != null ? (
+            <TableTitle>
+              {title ?? t.accessCodes}{' '}
+              <Caption>({filteredAccessCodes.length})</Caption>{' '}
```

Comment:
This nbsp is new not in the other one. Is it intentional?
```suggestion
              <Caption>({filteredAccessCodes.length})</Caption>
```